gamefixes-egs/umu-1361510.py

```diff
@@ -0,0 +1,2 @@
+../gamefixes-steam/1361510.py
+# EGS a4977aac967e4e2c827336f074c82d64
```

Comment:
To make this fix more accessible to users, add an entry for this title in the [umu-database](https://github.com/Open-Wine-Components/umu-database). Just be sure to follow our guidelines. See https://github.com/Open-Wine-Components/umu-database/blob/main/umu-database.csv for concrete examples

Reply:
ok. i'll do so

Reply:
done!